styles/blocks/footer/footer.less

```
@@ -0,0 +1,122 @@
+@footer-icons: instagram, twitter, vk, facebook;
+@footer-buttons: app-store, google-play;
+
+.footer {
+    background-color: #212122;
+    display: flex;
+    flex-direction: column;
+    padding: 40px 0 35px;
+
+    @media @screen-only-s {
+        display: flex;
+        padding: 30px 0 20px;
+    }
+
+    &__main {
+        display: flex;
+        align-items: flex-end;
+        padding: 0 0 30px;
+        border-bottom: 1px solid #666666;
+        margin: 0 0 30px;
+        width: 100%;
+
+        @media @screen-only-m {
+            justify-content: space-between;
+        }
+
+        @media @screen-only-s {
+            box-sizing: border-box;
+            flex-direction: column;
+            align-items: flex-start;
+            margin-bottom: 15px;
+        }
+    }
+
+    &__list {
+        list-style: none;
+        display: flex;
+        gap: 15px;
+        padding: 0;
+        margin: 0 0 10px 0;
+
+        @media @screen-only-s {
+            margin-bottom: 30px;
+        }
+
+        @media @screen-only-m {
+            margin: 0;
+        }
+    }
+
+    &__link {
+        width: 36px;
+        height: 36px;
+        display: block;
+        transition: opacity 0.3s ease-in-out;
+
+        &:hover {
+            opacity: 0.6;
+        }
+
+        each(@footer-icons, {
+            &_@{value} {
+                background-image: url('@{value}.svg');
+            }
+        })
+    }
+
+    &__buttons-container {
+        display: flex;
+        gap: 8px;
+        margin: 0 0 10px 130px;
+
+        @media @screen-only-m-s {
+            margin: 0;
+        }
+    }
+
+    &__button {
+        border: 0;
+        padding: 0;
+        width: 140px;
+        height: 40px;
+        background-color: transparent;
+        transition: opacity 0.3s ease-in-out;
+
+        &:focus {
+            outline: none;
+        }
+
+        &:hover {
+            cursor: pointer;
+            opacity: 0.6;
+        }
+
+        each(@footer-buttons, {
+            &_@{value} {
+                background-image: url('@{value}.svg');
+            }
+        })
+    }
+
+    &__logo {
+        margin-left: auto;
+
+        @media @screen-only-m-s {
+            display: none;
+        }
+    }
+
+    &__info {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        margin: 0 auto;
+    }
+
+    &__text {
+        color: #999999;
+        margin: 0;
+        line-height: 25px;
+    }
+}
```

добавь пустую строку в конец файла
```suggestion
}

```